services/video_chess_indexer/convert.py

```diff
@@ -3,11 +3,6 @@
 class Convert:
```

Comment:
LabelConverter or something like that? and add a class docstring explaining what its for. Ideally someone should be able to look at it and know whats going on. So docstrings++++ and descriptive names++++